src/schema/v2/sale/index.ts

```diff
@@ -207,16 +210,31 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
         description:
           "A formatted description of when the auction starts or ends or if it has ended",
         resolve: (
-          { start_at, end_at, ended_at, live_start_at },
+          {
+            start_at,
+            end_at,
+            ended_at,
+            live_start_at,
+            cascading_end_time_interval,
+          },
           _options,
           { defaultTimezone }
         ) => {
-          return formattedStartDateTime(
-            start_at,
-            ended_at || end_at,
-            live_start_at,
-            defaultTimezone || DEFAULT_TZ
-          )
+          if (cascading_end_time_interval) {
```

Comment:
I'm cool with using the different helpers for now based on whether `cascading_end_time_interval` is present, but noting that we should probably be formatting things the same way for all sales regardless of whether cascading end time is turned on. So ultimately, we should just use one helper and return the same thing in both cases.